### PR TITLE
clinician dashboard

### PR DESCRIPTION
**Pull request by philipcalderwood**
- Added diagnosis "Not Specified" to list of diagnoses. This would be the zero option to replace NULL.
- Removed clinician being able to change a patients name
- Used modifiable attributes to allow clinician to change a patients diagnosis
- Modified edit patient info logic for Patients table
- Fixed single *arg issue in edit patient info which had repearred somehow
- Some error checking which could be tidied
- **Updated `modules/patient.py` to include diagnosis in init**
- Added verbose exception to `modules/user.py` > `User` > `edit_info`
- Improved `wait_terminal()` to accept a message to display`

### CODE REVIEW

File: modules/patient.py [hunk 1]
```diff
@@ -38,6 +38,7 @@ def __init__(
         email: str,
         is_active: bool,
         clinician_id: int,
+        diagnosis: str = "NULL",
```
Comment (philipcalderwood) [inline modules/patient.py:41]: Change this to not specified

File: modules/user.py [hunk 2]
```diff
@@ -59,9 +59,9 @@ def edit_info(self, attribute: str, value: Any) -> bool:
             return True
 
         # If there is an error with the query
-        except sqlite3.OperationalError:
+        except sqlite3.OperationalError as e:
             print(
-                "Error updating, likely the selected attribute does not exist for Users"
+                f"{e} Error updating, likely the selected attribute does not exist for Users"
```
Comment (philipcalderwood) [inline modules/user.py:64]: Rewrite cleaner

File: modules/clinician.py [hunk 3]
```diff
@@ -279,30 +280,53 @@ def get_all_patients_by_diagnosis(self, diagnosis: str):
...
-    def edit_patient_info(self, patient: Patient):
-        """Edit patient information
-
-        WORK IN PROGRESS! -> User obj has no MODIFIABLE ATTRIBUTES
-        """
+    def edit_patient_info(self, patient: Patient) -> bool:
+        """Edit patient information"""
+        attribute: str
+        value: str
+        patient.MODIFIABLE_ATTRIBUTES.append("diagnosis")
```
Comment (davidve0206) [inline modules/clinician.py:287]: Note from meeting: this is not necessary as your are updating the DB directly

Comment (philipcalderwood) [reply modules/clinician.py:287]: ty

**Review by davidve0206 — APPROVED** [review]

**Pull request merged**